docs/_advanced/custom_routes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Adding Custom HTTP routes 
+lang: en
+slug: custom-routes
+order: 9
+---
+
+<div class="section-content">
+
+Adding custom HTTP routes is quite straightforward when using Bolt's built-in `ExpressReceiver`. Since `Bolt@v2.1.0`, `ExpressReceiver` added a `router` property, which exposes the Express [Router](http://expressjs.com/en/4x/api.html#router) on which more routes can be added.
```

Comment:
Just avoiding the use of `Bolt@...` since that confusingly looks like a package name i should be able to install from npm (whereas the actual package name is `@slack/bolt@...`). I'd be fine with using the whole package name too. I just suggested something a little shorter that got the same point across (hopefully).

```suggestion
Adding custom HTTP routes is quite straightforward when using Bolt's built-in `ExpressReceiver`. Since `v2.1.0`, `ExpressReceiver` added a `router` property, which exposes the Express [Router](http://expressjs.com/en/4x/api.html#router) on which more routes can be added.
```